### ISSUE TITLE
Regression: Environment mapping

### ISSUE THREAD

**Issue by DrMcCoy** [xoreos/xoreos#404]
Seems like @mirv-sillyfish shader renderer PR broke reflective environment mapping (with the old render path).

This is what I see in Chapter 2 of the original NWN campaign:
![20190114t223827](https://user-images.githubusercontent.com/156130/51148782-b76f5f80-185f-11e9-9867-99b51ccb3354.png)

Similarly for KotOR.

A quick git bisect pointed to 5bb1f00cf as the problem commit.

**Comment by DrMcCoy** [comment]
For posterity, a (reduced) log of @mirv-sillyfish talking on IRC about this issue:

```
12:03:19 < mirv_> so basically it works, but probably hasn't assigned the env map
12:03:34 < mirv_> or otherwise thinks there isn't one when there should be
12:05:09 < mirv_> ModelNode::getEnvironmentMap is my immediate thought
12:05:24 < mirv_> second is it has to be somewhere in renderGeometryNormal
12:06:36 < mirv_> shader path renders ok?
12:06:48 <@DrMcCoy> Haven't tried
12:10:26 < mirv_> ok, think I see the problem....renderGeometryEnvMappedOver/Under don't look to be called?!
12:10:48 <@DrMcCoy> That would be bad :P
12:11:31 <@DrMcCoy> Might be that it accidentally got changed during a conflict resolution rebase
12:11:58 < mirv_> eh, relatively minor fix - I think I see where
12:13:26 < mirv_> yep, it's likely the change made to "renderGeometry"
12:14:25 < mirv_> something in there, need to just call the renderGeometryEnvMapped* functions before/after renderGeometryNormal
```

**Comment by DrMcCoy** [comment]
Fixed with #417 :)
